src/components/Header/Header.module.css

```diff
@@ -16,26 +16,35 @@
   z-index: 10;
 }
 .logo {
-  width: 160px;
+  width: 100px;
 }
-nav {
+.header-nav {
   width: 1000px;
   max-width: 750px;
   margin-right: 60px;
   display: flex;
   align-items: center;
   justify-content: space-between;
 }
-nav ul {
+.nav-list {
   display: flex;
   align-items: center;
 }
 
-nav ul li {
+.nav-list-item{
   list-style: none;
   color: var(--primary-white);
   padding-right: 30px;
 }
+.nav-link {
+  color: var(--primary-white);
+  text-decoration: none;
+  /* gets rid of the underline */
```

Comment:
I would either put this comment at the end of line 41 (clearly refers to the `text-decoration` rule) or else remove the comment. I had it in the sample code to be obvious about what it was doing, in case that was unfamiliar, but once it's on its own line, the odds of it getting separated from the relevant line or becoming unclear about what it refers to go way up.